includes/helper-functions.php

```
@@ -1120,3 +1044,110 @@ function edac_check_if_post_id_is_woocommerce_checkout_page( $post_id ) {
+	$dom->loadHTML(
+		htmlspecialchars_decode( $html, ENT_QUOTES ),
+		LIBXML_HTML_NOIMPLIED | LIBXML_HTML_NODEFDTD
+	);
+	
+	$xpath = new \DOMXPath( $dom );
+	
+	// Check for img elements first.
+	$img_elements = $xpath->query( '//img' );
+	if ( $img_elements->length > 0 ) {
+		foreach ( $img_elements as $element ) {
+			$src = $element->getAttribute( 'src' );
```

PHPStan (2.0.3)

Call to an undefined method DOMNode::getAttribute().


(method.notFound)